docker/1.1.0/base/Dockerfile.gpu

```diff
@@ -0,0 +1,35 @@
+FROM nvidia/cuda:9.0-cudnn7-runtime-ubuntu16.04
```

Comment:
I think we already covered this question: PyTorch 1.1.0 is compatible with CUDA 9.0, right?

Reply:
Yes, we did :)